### PR TITLE
Add Button component

### PR DESCRIPTION
**Pull request by aronstrandberg**
Add a Button component to simplify dealing with different classes and shit

### CODE REVIEW

File: app/components/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  onClick,
+  children,
+}) {
+  const classes = classNames({
+    [styles[`${color}`]]: color,
+    [styles.disabled]: disabled,
+    [styles.full]: full,
+  }, styles.button, className)
+  return (
+    <Wrapper display={wrapper}>
```
Comment (aronstrandberg) [inline app/components/Button/index.js]: anyone's got a smarter way of doing this?

File: app/components/Button/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      <button type={type} className={classes} onClick={onClick}>
+        { children }
+      </button>
+    </Wrapper>
+  )
+}
+
+const Wrapper = ({ display, children }) => {
+  if (!display) return children
```
Comment (aronstrandberg) [inline app/components/Button/index.js]: anyone's got a smarter way of doing this?

File: app/components/Button/index.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+Wrapper.propTypes = {
+  display: PropTypes.bool,
+  children: PropTypes.node.isRequired,
+}
+
+Button.propTypes = {
+  className: PropTypes.string,
+  type: PropTypes.oneOf([
```
Comment (jeppes) [inline app/components/Button/index.js]: nice!

File: app/containers/CvEdit/styles.css [hunk 4]
```diff
@@ -21,10 +21,10 @@
 }
 
 .cvAction {
-  text-transform: uppercase;
-  font-weight: 700;
+  /*text-transform: uppercase;*/
```
Comment (jeppes) [inline app/containers/CvEdit/styles.css]: remove comments

Comment (aronstrandberg) [reply app/containers/CvEdit/styles.css]: rip

**Pull request merged**